templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -250,5 +298,19 @@ export default {
       }
     }
   }
+
+  .menu-wrapper {
+    display: flex;
+    flex-direction: column;
+    margin-top: 8px;
+    margin-bottom: 8px;
+  }
+
+  .menu {
+    z-index: 100;
+    position: relative;
+    float: none;
+    background: white;
+  }

```

Comment:
I suggest looking into using Bootstrap's vertical nav ([Bootstrap-Vue component](https://bootstrap-vue.org/docs/components/nav#vertical-variation)) instead of having to custom create it like this.